craft_parts/plugins/plugins.py

```diff
@@ -116,6 +116,12 @@ def register(plugins: Dict[str, PluginType]) -> None:
     _PLUGINS.update(plugins)
 
 
+def unregister(plugins: List[str]) -> None:
```

Comment:
It would be nice to be able to say `unregister("my-plugin")` without having to put a list. One option:

```suggestion
def unregister(*plugins: str) -> None:
```

Reply:
Unregistering all plugins (including built-in):

```python
unregister(*get_registered_plugins().keys())
```